pkg/virt-launcher/virtwrap/cli/libvirt.go

```diff
@@ -264,7 +265,29 @@ func (l *LibvirtConnection) GetAllDomainStats(statsTypes libvirt.DomainStatsType
 	}
 	return domStats, nil
 }
-
+func (l *LibvirtConnection) GetQemuVersion() (string, error) {
+	version, err := l.Connect.GetVersion()
+	// The following code works because version it's an uint32 var. Therefore, the divisions will result in
+	// an integer number. For instance:
+	// version = 7002002
+	// major = 7002002 / 1000000 = 7 --> decimals are discard in this operation
+	// version = 7002002 - (7*1000000) = 2000
+	// minor = 2002 / 1000 = 2
+	// version = version - (2*1000) = 2
+	// release = 2
+
+	major := version / 1000000
+	version = version - (major * 1000000)
+	minor := version / 1000
+	version = version - (minor * 1000)
+	release := version
+
+	processedVersion := fmt.Sprintf("QEMU %d.%d.%d", major, minor, release)
+	if err != nil {
```

Comment:
Move the err handling to L270 and directly return `fmt.Sprintf("QEMU %d.%d.%d", major, minor, release), nil`?

Reply:
Addressed.